custom/enikshay/private_sector_datamigration/management/commands/update_names.py

```diff
@@ -0,0 +1,55 @@
+from django.core.management import BaseCommand
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.users.models import CommCareUser
+from custom.enikshay.private_sector_datamigration.models import UserDetail, Agency
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+
+    def handle(self, domain, **options):
+        locs_matching_type = SQLLocation.active_objects.filter(
+            domain=domain,
+            location_type__code__in=[
+                'pac',
+                'pcp',
+                'plc',
+                'ps-fieldstaff',
+            ],
+        )
+
+        total_to_process = locs_matching_type.count()
+
+        for i, loc in enumerate(locs_matching_type):
+            private_sector_agency_id = loc.metadata.get('private_sector_agency_id', '')
+            if not private_sector_agency_id.isdigit():
+                continue
+            agency_id = int(private_sector_agency_id)
+            try:
+                agency = Agency.objects.get(agencyId=agency_id)
+            except Agency.DoesNotExist:
+                continue
+            loc.name = "%s - %d" % (agency.agencyName, agency_id)
```

Comment:
👍 
I think you could also have just set this as `"{} - {}".format(loc.name, agency_id)`, but your way is good because it will also filter out all non-migrated agencies.